tests/tests_unit/test_cdf_tk/test_load.py

```diff
@@ -507,17 +527,16 @@ def test_loader_takes_list(
             loaded, (loader.resource_write_cls, loader.list_write_cls)
         ), f"loaded must be an instance of {loader.list_write_cls} or {loader.resource_write_cls} but is {type(loaded)}"
 
-    def check_url(self, url) -> bool:
+    @staticmethod
+    def check_url(url) -> bool:
         try:
             response = requests.get(url, allow_redirects=True)
             return response.status_code >= 200 and response.status_code <= 300
         except requests.exceptions.RequestException:
             return False
 
-    @pytest.mark.parametrize("Loader", sorted(find_subclasses(ResourceLoader), key=lambda x: x.folder_name))
-    def test_loader_has_doc_url(
-        self, Loader: type[ResourceLoader], cdf_tool_config: CDFToolConfig, monkeypatch: MonkeyPatch
-    ):
+    @pytest.mark.parametrize("Loader", ALL_LOADERS)
```

Comment:
@ronpal I switched this to test all loaders, not just the resource loaders, and thus ended up adding the extra URLs.